src/Cache/FileCacheManager.php

```diff
@@ -32,6 +32,8 @@
  */
 final class FileCacheManager implements CacheManagerInterface
 {
+    public const WRITE_FREQUENCY = 10;
```

Comment:
Wouldn't it be better to have it dynamic (configurable)? Then it would be easier to test it, but also possible to fine-tune through config?

Reply:
for tests - I would suggest to go this part when we would have the good idea for test here.

for fine-tuning - I didn't see the need for that part, actually. I followed YAGNI here. happy to have this configurable if the needs pop up